cnd/cnd.debugger.common2/licenseinfo.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <!-- Tests expect this file without an ASF License header -->
+        <file>test/unit/data/goldenfiles/org/netbeans/modules/cnd/debugger/common2/debugger/breakpoints/BreakpointsCodecTest/Breakpoints_new.xml</file>
+        <file>test/unit/data/goldenfiles/org/netbeans/modules/cnd/debugger/common2/debugger/breakpoints/BreakpointsCodecTest/Breakpoints_old.xml</file>
+        <file>test/unit/data/goldenfiles/org/netbeans/modules/cnd/debugger/common2/debugger/breakpoints/BreakpointsCodecTest/Breakpoints_saved.xml</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="TEMPLATE_MINIMAL_IP" />
```

Comment:
These are not really templates. I suggest to add a new entry to the `COMMENT_TYPE` enum like this:

```diff
# This patch file was generated by NetBeans IDE
# It uses platform neutral UTF-8 encoding and \n newlines.
--- a/nbbuild/antsrc/org/netbeans/nbbuild/extlibs/licenseinfo/CommentType.java
+++ b/nbbuild/antsrc/org/netbeans/nbbuild/extlibs/licenseinfo/CommentType.java
@@ -22,7 +22,8 @@
     TEMPLATE_MINIMAL_IP("The file is used as a template and contains minimal IP."),
     CATEGORY_B("The contained code is used at runtime and must be included in source form."),
     COMMENT_UNSUPPORTED("Filetype does not support comments."),
-    GUI_USABILITY("Code is visible at runtime and added license would negatively affect usability.");
+    GUI_USABILITY("Code is visible at runtime and added license would negatively affect usability."),
+    TEST_DATA("The file is used in tests and license header can't be added.");
     
     private String outputComment;
     
```